spec/tree-view-spec.coffee

```diff
@@ -394,27 +394,52 @@ describe "TreeView", ->
         atom.project.setPaths([rootDirPath])
         treeView.height(100)
         jasmine.attachToDOM(workspaceElement)
+      describe "if alphanumericSort is set to true", ->
```

Comment:
Rather than testing this as a dimension of the `reveal-active-file` command, I think it probably warrants a new top-level test like the following:

```coffee
describe "the tree-view.naturalSort option", ->
  it "determines whether tree view items are sorted naturally or in strict alphanumeric order", ->
    # load up the tree view with an example that sorts differently based on the option
    # start with the option one way, assert the order
    # toggle the option, assert the order again
```

And leave it at that. Feel free to interpret my suggested test titles a bit. It's just a rough example.